x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/actions/update_actions.ts

```diff
@@ -5,205 +5,245 @@
       });
 
-      it('@skipInQA should be able to create a new webhook action and attach it to a rule with a meta field and run it correctly', async () => {
-        const hookAction = await createNewAction(supertest, log);
-        const rule = getSimpleRule();
-        await createRule(supertest, log, rule);
-        const ruleToUpdate: RuleCreateProps = {
-          ...getRuleWithWebHookAction(hookAction.id, true, rule),
-          meta: {}, // create a rule with the action attached and a meta field
+      it('expects an updated rule with a webhook action runs successfully', async () => {
+        const webhookAction = await createWebHookRuleAction(supertest);
+
+        await supertest
+          .post(DETECTION_ENGINE_RULES_URL)
+          .set('kbn-xsrf', 'true')
+          .set('elastic-api-version', '2023-10-31')
+          .send(getCustomQueryRuleParams({ rule_id: 'rule-1' }))
+          .expect(200);
+
+        const ruleAction = {
+          group: 'default',
+          id: webhookAction.id,
+          params: {
+            body: '{}',
+          },
+          action_type_id: '.webhook',
         };
-        const updatedRule = await updateRule(supertest, log, ruleToUpdate);
+        const updatedRule = await updateRule(
+          supertest,
+          getCustomQueryRuleParams({
+            rule_id: 'rule-1',
+            actions: [ruleAction],
+            enabled: true,
+          })
+        );
+
         await waitForRuleSuccess({ supertest, log, id: updatedRule.id });
-      });
 
-      it('@skipInQA should not change properties of immutable rule when applying actions to it', async () => {
-        // actions and throttle to be removed from assertion (it asserted in a separate test case)
-        const actionsProps = ['actions', 'throttle'];
+        const { body } = await supertest
+          .get(DETECTION_ENGINE_RULES_URL)
+          .set('kbn-xsrf', 'true')
+          .set('elastic-api-version', '2023-10-31')
+          .query({ id: updatedRule.id })
+          .expect(200);
 
-        const immutableRule = await getImmutableRule();
-        const hookAction = await createNewAction(supertest, log);
-        const ruleToUpdate = getRuleWithWebHookAction(
-          hookAction.id,
-          immutableRule.enabled,
-          ruleToUpdateSchema(immutableRule)
-        );
-        const updatedRule = await updateRule(supertest, log, ruleToUpdate);
-        const expected = omit(removeServerGeneratedProperties(updatedRule), actionsProps);
+        expect(body?.execution_summary?.last_execution?.status).toBe('succeeded');
+      });
 
-        const immutableRuleToAssert = {
-          ...omit(removeServerGeneratedProperties(immutableRule), actionsProps),
-          revision: 1, // Unlike `version` which is static for immutable rules, `revision` will increment when an action/exception is added
+      it('expects an updated rule with a webhook action and meta field runs successfully', async () => {
```

Comment:
This one is skipped in QA. Was it intentionally enabled?

Reply:
Tbh I wonder why this test was skipped in QA. I added `@skipInQA` tag back but it looks like the tests should work in QA environment.